python/cdp/evm_server_account.py

```diff
@@ -325,6 +326,26 @@ async def request_faucet(
             token,
         )
 
+    async def sign_typed_data(
```

Comment:
Does smart account have a `sign_typed_data`

Reply:
i didn't think so because the yaml only has it under evm/accounts, unless that may also include smart accounts